docs/user-guide/03-convenience-features.md

```diff
@@ -262,6 +264,50 @@ int test()
 }
 ```
 
+Slang will also implicitly call a *default* constructor of all parents of a derived struct (same as C++):
+```csharp
+struct MyType_Base
+{
+    int myVal1;
+    __init() {myVal1 = 22;}
+}
+
+struct MyType1 : MyType_Base
+{
+    int myVal2;
+    __init()
+    {
+        // implicitly calls `MyType_Base::__init()`
+        myVal2 = 15;
+    }
+}
+testMyType1()
```

Comment:
void testMyType1()

Reply:
I don't think this comment is addressed. And there is one more function with the same mistake.

Reply:
hmm, I didn't see any review comments (or an email) when I looked?